build_scripts/entrypoint.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# docker entrypoint script.
```

Comment:
when is this script executed? What does this do? please add a 1-2 liner brief about it.

Reply:
@rathorevaibhav this file is being from `Dockerfile` called while building the app for production using docker. It basically checks if DB connection is established, run the migration and at last starts the application.

Reply:
Added brief comment.